src/Microsoft.Azure.SignalR.Common/Exception/AccessTokenTooLongException.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.Azure.SignalR.Common
+{
+    public class AccessTokenTooLongException : Exception
```

Comment:
`[Serializable]` exception, a sample https://github.com/aspnet/SignalR/blob/master/src/Microsoft.AspNetCore.SignalR.Common/HubException.cs#L13. 
As it is `public`, make the naming more specific, e.g. `AzureSignalRAccessTokenTooLongException`?

Reply:
Get it. Do we need to add summary comments as well?

---
In reply to: [243783574](https://github.com/Azure/azure-signalr/pull/303#discussion_r243783574) [](ancestors = 243783574)